ouroboros-consensus/ouroboros-consensus.cabal

```diff
@@ -311,6 +311,8 @@ library
                      , ouroboros-network-framework
                      , ouroboros-network
 
+                     , canonical-json
+                     , aeson
```

Comment:
It's a bit annoying to add these deps. This is used in the following commit to serialize `ShelleyGenesis` and `GenesisData`. We could instead attempt to create orphan instances.